ludwig/api.py

```diff
@@ -930,10 +934,12 @@ def experiment(
             `inf_or_nan` checks.
 
         # Return
-        :return: (Tuple[dict, dict, tuple, str)) `(test_results, train_stats, preprocessed_data, output_directory)`
-            `test_results` dictionary with
-            overall statistcs on the test_set, `train_stats` dictionary with
-            overall training statistics,
+        :return: (Tuple[dict, dict, tuple, str)) `(evaluation_results, training_stats, preprocessed_data, output_directory)`
+            `evaluation_results` dictionary with evaluation performance
+                statistics on the test_set,
+            `training_stats` is a dictionary of training statistics for
```

Comment:
training_stats -> training_stats
In general I prefer more explicit names in the docs, in the code we can name them differently for convenience and increasing visibility, but here clarity is the main concern

Reply:
done